src/js/util.js

```diff
@@ -476,11 +476,30 @@ var Util;
                 return;
             }
 
-            var list = element.parentElement;
+            var parent = element.parentElement;
 
-            if (list.parentElement.nodeName.toLowerCase() === 'p') { // yes we need to clean up
-                this.unwrap(list.parentElement, ownerDocument);
+            // do we need to fix list nesting? (valid is: ul/ol > li > ul/ol )
+            if ((parent.nodeName.toLowerCase() === 'ul' || parent.nodeName.toLowerCase() === 'ol') && (parent.parentElement.nodeName.toLowerCase() === 'ul' || parent.parentElement.nodeName.toLowerCase() === 'ol')) {
+                // does a previous list item exist? (a second order list needs to be inside a previous list item)
+                if (parent.previousElementSibling.nodeName.toLowerCase() === 'li') {
```

Comment:
This reference to `parent.previousElementSibling` can be null, which causes an error here.